Sporadic test failure (6x): KHttpClientTest

```
java.net.SocketTimeoutException: Read timed out
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1969)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1964)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1963)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1531)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1515)
	at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)
	at khttp.responses.GenericResponse.openRedirectingConnection$khttp(GenericResponse.kt:127)
	at khttp.responses.GenericResponse.getConnection(GenericResponse.kt:163)
	at khttp.responses.GenericResponse.getRaw(GenericResponse.kt:207)
	at khttp.responses.GenericResponse.getContent(GenericResponse.kt:216)
	at khttp.responses.GenericResponse.init$khttp(GenericResponse.kt:350)
	at khttp.KHttp.request(KHttp.kt:59)
	at khttp.KHttp.request$default(KHttp.kt:57)
	at KHttpClientTest.doRequest(KHttpClientTest.groovy:23)
	at io.opentelemetry.auto.test.base.HttpClientTest.basic #method request #url - tagQueryString=#tagQueryString_closure4(HttpClientTest.groovy:90)
	at io.opentelemetry.auto.test.base.HttpClientTest.basic #method request #url - tagQueryString=#tagQueryString_closure4(HttpClientTest.groovy)
	at io.opentelemetry.auto.test.utils.ConfigUtils.withConfigOverride(ConfigUtils.groovy:42)
	at io.opentelemetry.auto.test.base.HttpClientTest.basic #method request #url - tagQueryString=#tagQueryString(HttpClientTest.groovy:89)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/java.io.BufferedInputStream.fill(BufferedInputStream.java:252)
	at java.base/java.io.BufferedInputStream.read1(BufferedInputStream.java:292)
	at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:351)
	at java.base/sun.net.www.http.HttpClient.parseHTTPHeader(HttpClient.java:754)
	at java.base/sun.net.www.http.HttpClient.parseHTTP(HttpClient.java:689)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1610)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1515)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3117)
	at khttp.responses.GenericResponse$Companion.getCookieJar$khttp(GenericResponse.kt:35)
	at khttp.responses.GenericResponse$Companion$defaultEndInitializers$3.invoke(GenericResponse.kt:116)
	at khttp.responses.GenericResponse$Companion$defaultEndInitializers$3.invoke(GenericResponse.kt:32)
	at khttp.responses.GenericResponse$connection$2.invoke(GenericResponse.kt:164)
	at khttp.responses.GenericResponse$connection$2.invoke(GenericResponse.kt:30)
	at khttp.responses.GenericResponse.openRedirectingConnection$khttp(GenericResponse.kt:124)
	... 11 more
```

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1314/workflows/2ad34987-b0fe-4dc0-9cee-5c5b40c70be7/jobs/10701/tests

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1407/workflows/435b9f2f-b54f-4e6f-8406-845bca2440e6/jobs/11356/tests

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1420/workflows/fdb7f15c-ef80-4744-803c-238797264dd0/jobs/11412/tests

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1419/workflows/b35fcaa0-20de-43a7-b7d0-2a02a82313b4/jobs/11403/tests

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1558/workflows/ac0de63a-4fa8-43ab-baba-cc616717feae/jobs/12260/tests

--- a/instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy
+++ b/instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy
@@ -22,8 +22,8 @@ class KHttpClientTest extends HttpClientTest {
   @Override
   int doRequest(String method, URI uri, Map<String, String> headers, Closure callback) {
     // khttp applies the same timeout for both connect and read
-    def timeout = Math.max(CONNECT_TIMEOUT_MS, READ_TIMEOUT_MS)
-    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, timeout)
+    def timeoutSeconds = Math.max(CONNECT_TIMEOUT_MS, READ_TIMEOUT_MS) / 1000
+    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, timeoutSeconds)
     if (callback != null) {
       callback.call()
     }

--- a/instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy
+++ b/instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy
@@ -21,7 +21,9 @@ class KHttpClientTest extends HttpClientTest {
 
   @Override
   int doRequest(String method, URI uri, Map<String, String> headers, Closure callback) {
-    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, 1)
+    // khttp applies the same timeout for both connect and read
+    def timeout = Math.max(CONNECT_TIMEOUT_MS, READ_TIMEOUT_MS)
+    def response = KHttp.request(method, uri.toString(), headers, Collections.emptyMap(), null, null, null, null, timeout)
     if (callback != null) {
       callback.call()
     }

